app/models/category.rb

```diff
@@ -0,0 +1,11 @@
+class Category < ApplicationRecord
+
+    belongs_to :admin_user
+
+    validates :admin_user, presence: true
```

Comment:
No need to add these lines of code because Category has no direct relationship to User, based on our ERD
![image](https://user-images.githubusercontent.com/35232348/115405590-f8196880-a220-11eb-944f-bb5eef20a199.png)

Reply:
Thank you for correct me. I could make "Id1 title:math description:lesson for math" in Categories DB.